### PR TITLE
update for employee names and income amount

### PR DESCRIPTION
**Pull request by katedee**
- add the employee name/display the employee name
- update the values in incomeSources

### CODE REVIEW

File: locales/en.json [hunk 1]
```diff
@@ -255,5 +255,8 @@
   "Total Deducted (437)": "Total Deducted (437)",
   "Total Credits (482)": "Total Credits (482)",
   "Total Refund (484)": "Total Refund (484)",
-  "I confirm that, as far as I know, all the information in this tax return is accurate": "I confirm that, as far as I know, all the information in this tax return is accurate"
-}
+  "I confirm that, as far as I know, all the information in this tax return is accurate": "I confirm that, as far as I know, all the information in this tax return is accurate",
+  "undefined Total Income": "undefined Total Income",
```
Comment (pcraig3) [inline locales/en.json]: This key means that it gets "undefined" at some point. is that happening during a test?

Comment (katedee) [reply locales/en.json]: so when we change/update the data, this'll happen, unless you go back to the start and enter an access code, to force it to pull in fresh user info

File: views/financial/income.pug [hunk 2]
```diff
@@ -29,7 +29,7 @@ block content
             th(colspan="2").breakdown-table__subheading #{__(`Detailed breakdown of income earned in ${year}`)}
           each income in incomeSources
             tr
-              td #{__(`Employer ${count} Total Income`)}
+              td #{__(`${income.name} Total Income`)}
```
Comment (pcraig3) [inline views/financial/income.pug]: Actually, let's only translate the string "Total Income" instead of including the employer name

**Comment by pcraig3** [comment]
One final comment just to note that we are talking about "employer" names.

**Pull request merged**